fr.rubet.rpn.yml

```diff
@@ -0,0 +1,16 @@
+app-id: fr.rubet.rpn
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: rpn
+modules:
+  - mpfr.yaml
+  - name: rpn
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/louisrubet/rpn.git
+        tag: v2.4
+        commit: bafc48ab85e06003d38b4cba1a95fcbc4562acd5
+    post-install:
```

Comment:
there is no appstream fiel nor desktop file.

You never install the icons (only 128x128 or scalable SVG is required)

Reply:
you are super reactive! :+1: 
on my way

Reply:
> there is no appstream fiel
What do you mean by `no appstream`? Is my app ID wrong?

> nor desktop file.
This app is a cli, do I need to add a desktop file?

Reply:
(found it in the doc)